pkg/controller/machine.go

```diff
@@ -161,6 +161,12 @@ func (c *controller) reconcileClusterMachine(machine *v1alpha1.Machine) error {
 		return err
 	}
 
+	// Populate the controller configurations if there are any.
```

Comment:
This is setting it as a global flag. So if two machines are being reconciled simultaneously with different values. The values should get mixed up. This can be tested with a simple test. 

I think this values should be fetched during usage and not set here at a global level.

Reply:
Sure, I'll implement the getMachineDrainTimeOut() sort of methods for all configuration, thanks for the catch.